modules/updating-sno.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster.adoc
+
+= Updating single node {product-title}
+
+You can update, or upgrade, a single-node {product-title} cluster using either the console or CLI. 
+
+However, note the following: 
+
+* The prerequisite to pause the `MachineHealthCheck` resources is not required, because there is no other node to perform the health check.
```

Comment:
```suggestion
* The prerequisite to pause the `MachineHealthCheck` resources is not required because there is no other node to perform the health check.
```